dropgz/pkg/embed/payload.go

```diff
@@ -86,6 +87,17 @@ func deploy(src, dest string) error {
 		return err
 	}
 	defer rc.Close()
+	if _, err := os.Stat(dest); err == nil || !os.IsNotExist(err) {
+		old_file_dest := dest + OLD_FILE_SUFFIX
```

Comment:
Same deal here, this should be camel case: `oldFileDest`. But really, you can probably just go with `oldDest` since `dest` is its companion.

Reply:
Okay sure.